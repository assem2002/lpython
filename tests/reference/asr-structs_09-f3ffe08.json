{
    "basename": "asr-structs_09-f3ffe08",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/structs_09.py",
    "infile_hash": "06fb179f0c0c41124ffc960676a9f32cfe112ab4e39c6f6a7cb2a6e3",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-structs_09-f3ffe08.stderr",
    "stderr_hash": "295600c377d777d38b373e302e4a35f5bf19fc7f3ce4690039f10926",
    "returncode": 2
}