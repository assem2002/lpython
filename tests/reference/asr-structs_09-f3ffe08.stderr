semantic error: read not present in StringIO dataclass
  --> tests/errors/structs_09.py:13:23
   |
13 |     bytes_read: i32 = fd.read()
   |                       ^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
